### PR TITLE
FC-0057: add makefile command for black formatting

### LINKED ISSUE #31: Make sure code is properly formatted by black in CI
The check to implement in CI is something along the lines of:

    black --check forum/ tests/ test_utils/

Then, to format it, run:

    black forum/ tests/ test_utils/

### PR DESCRIPTION
**Pull request by Faraz32123**
- `make test-format` command view the files under forum/ tests/ test_utils/ directories that needs formatting
- `make format` command format all files under forum/ tests/ test_utils/ directories
- add black to dev requirements
- close #31

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -84,6 +87,9 @@ test-isort: ## run isort checks
 test-mypy: ## run type tests
 	mypy ${SRC_FILES_PROD}
 
+test-format: ## Run code formatting tests
```
Comment (regisb) [inline Makefile:90]: Please add this target to `make test-quality`, such that it is run in CI.

**Review by regisb — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (Faraz32123) [reply Makefile:90]: Sure.

**Comment by Faraz32123** [comment]
I haven't added black in requirements yet, that's why tests are failing.

File: requirements/dev.in [hunk 2]
```diff
@@ -5,5 +5,6 @@
 -r quality.txt            # Core and quality check dependencies
 -r ci.txt                 # dependencies for setting up testing in CI
 
+black                     # code formatter
```
Comment (regisb) [inline requirements/dev.in:8]: This should go in quality.in. CI installs quality.in, not dev.txt.

**Review by taimoor-ahmed-1 — APPROVED** [review]

**Pull request merged**